spec/attributes.md

```diff
@@ -1,846 +1,29 @@
-﻿# Attributes
+# Attributes
+
+This content has moved to the [`dotnet/csharpstandard`](https://github.com/dotnet/csharpstandard) repository.
+The list below provides links to each heading in this section. The links specify the C# 6 branch, which is version when the specifications merged.
+
+- <a id="attributes"></a>[Attributes](https://github.com/dotnet/csharpstandard/blob/draft-v6/standard/attributes.md#21-attributes)
```

Comment:
I'm a bit curious why we put the new docs into "draft-v6" folder. Why not just "v6" or no subfolder? Having "draft" means we'll have to fix a bunch of links once the docs are no longer draft.

Reply:
@jcouv "draft-v6" is the branch name. When we submit to ECMA, we'll change the name to "standard-v6", and GitHub will automatically update the links.